scripts/packages/pyheron/setup.py

```diff
@@ -0,0 +1,34 @@
+"""Pyheron setuptools based setup module"""
+import os
+from setuptools import setup, find_packages
+
+here = os.path.abspath(os.path.dirname(__file__))
+requirements = ['colorlog==2.6.1',
+                'protobuf==2.5.0',
+                'setuptools==18.0.1']
+
+setup(
+  name='pyheron',
+  version='0.0.1',
+  description='Python API for Heron topology',
+  long_description='Long description about pyheron',
```

Comment:
Pyheron package allows a developer to write python topology in Heron. Pyheron is backward compatible with the popular python API called streamparse. It can be run in various clusters including Mesos/Aurora, Mesos/Marathon, YARN, etc.